examples/jupyter/pincell_depletion.ipynb

```diff
@@ -0,0 +1,1028 @@
+      "    <nuclide ao=\"0.5145\" name=\"Zr90\" />\r\n",
+      "    <nuclide ao=\"0.1122\" name=\"Zr91\" />\r\n",
+      "    <nuclide ao=\"0.1715\" name=\"Zr92\" />\r\n",
+      "    <nuclide ao=\"0.1738\" name=\"Zr94\" />\r\n",
+      "    <nuclide ao=\"0.028\" name=\"Zr96\" />\r\n",
+      "  </material>\r\n",
+      "  <material id=\"3\" name=\"water\">\r\n",
+      "    <density units=\"g/cc\" value=\"1.0\" />\r\n",
+      "    <nuclide ao=\"0.999621\" name=\"O16\" />\r\n",
+      "    <nuclide ao=\"0.000379\" name=\"O17\" />\r\n",
+      "    <nuclide ao=\"1.99968852\" name=\"H1\" />\r\n",
+      "    <nuclide ao=\"0.00031148\" name=\"H2\" />\r\n",
+      "    <sab name=\"c_H_in_H2O\" />\r\n",
+      "  </material>\r\n",
+      "</materials>\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "materials.export_to_xml()\n",
+    "!cat materials.xml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Setting up for depletion\n",
+    "\n",
+    "The OpenMC depletion interface can be accessed from the `openmc.deplete` module, and has a variety of classes that will help us."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import openmc.deplete"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to run the depletion calculation we need the following information:\n",
+    "\n",
+    "1. Nuclide decay, fission yield, and reaction data\n",
+    "2. Operational power or power density\n",
+    "3. Desired depletion schedule\n",
+    "4. Desired time integration scheme\n",
+    "\n",
+    "The first item is necessary to determine the paths by which nuclides transmute over the depletion simulation. This includes spontaneous decay, fission product yield distributions, and nuclides produced through neutron-reactions. For example,\n",
+    "* Te129 decays to I129 with a half life of ~70 minutes\n",
+    "* A fission event for U-235 produces fission products like Xe135 according to a distribution\n",
+    "* For thermal problems, Am241 will produce metastable Am242 about 8% of the time during an $(n,\\gamma)$ reaction. The other 92% of capture reactions will produce ground state Am242\n",
+    "\n",
+    "These data are often distributed with other nuclear data, like incident neutron cross sections with ENDF/B-VII. The `openmc.deplete.Chain` object is responsible for representing this information. Two executables, `openmc-make-depletion-chain` and `openmc-make-depletion-chain-casl` will read in nuclear data using the `OPENMC_ENDF_DATA` environment variable and produce two depletion chain files. The first is a more detailed file containing every reaction and fission product possible, while the second uses data from CASL [Specification for the VERA Depletion Benchmark Suite](https://doi.org/10.2172/1256820) report to reduce the number of isotopes.\n",
```

Comment:
I think what we ought to do is just have depletion chains available on openmc.org and point the user there rather than confuse them with the discussion of these scripts, environment variables, etc.

Reply:
That would be super helpful. How would we go about doing that?

Reply:
Link to `openmc.org/depletion-chains` included in cb92f73